docs/examples/800_istio.adoc

```diff
@@ -0,0 +1,169 @@
+= Istio Support
+
+== Overview
+
+Coherence Operator 3.1.15 and later supports Istio 1.9.1 and later. You can run the operator and Coherence cluster managed by the operator with Istio sidecar injection enabled.  Coherence caches can be accessed from outside via Coherence*Extend, REST, and other supported Coherence clients.  
+
+== Limitations
+
+The current support for Istio has the following limitation:
+
+ * TLS between cluster members do not work in Istio environment. Coherence client can support TLS through Istio secure gateway with TLS termination to connect to Coherence cluster running inside kubernetes.  For example, you can apply the following Istio gateway and and virtual service in the namespace of the Coherence cluster:
+
+tlsGateway.yaml
+[source,bash]
+----
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: tlsgateway
+spec:
+  selector:
+    istio: ingressgateway # use istio default ingress gateway
+  servers:
+  - port:
+      number: 8043
+      name: tls
+      protocol: TLS
+    tls:
+      mode: SIMPLE
+      credentialName: "extend-credential" # must be the same as secret for the TLS key and cert
+      maxProtocolVersion: TLSV1_3
+    hosts:
+    - "*"
+----
+
+
+tlsVS.yaml
+[source,bash]
+----
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: extend
+spec:
+  hosts:
+  - "*"
+  gateways:
+  - tlsgateway
+  tcp:
+  - match:
+    route:
+    - destination:
+        port:
+          number: 20000                    # the proxy server listening port
+        host: example-cluster-proxy-proxy  # the ClusterIP service name of the proxy server
+----
+
+Then a Coherence*Extend client can connect to the proxy server via TLS protocol at port 8043.
```

Comment:
How? An example of the Coherence client configuration might be nice or at least something to point customers to how to configure a client to use TLS.